Modules/Scripted/DICOMDirectReader/DICOMDirectReader.py

```diff
@@ -0,0 +1,68 @@
+import os
+import slicer
+from slicer.i18n import tr as _
+from slicer.i18n import translate
+from slicer.ScriptedLoadableModule import ScriptedLoadableModule
+
+
+class DICOMDirectReader(ScriptedLoadableModule):
+    """A module for loading Revvity Living Image files."""
+
+    def __init__(self, parent):
+        """Initialize this class."""
+        ScriptedLoadableModule.__init__(self, parent)
+        self.parent.title = _("DICOMDirectReader")
+        self.parent.categories = ["", translate("qSlicerAbstractCoreModule", "Informatics")]  # top level module
+        parent.dependencies = ["DICOM"]
+        parent.contributors = [""]
+        parent.helpText = _("This module allows direct loading of DICOM files without importing into the DICOM database.")
+        parent.acknowledgementText = _("This work is supported by NA-MIC, NAC, BIRN, NCIGT, and the Slicer Community.")
+        # don't show this module - it is only for registering a reader
+        parent.hidden = True
+        self.parent = parent
+
+
+class DICOMDirectReaderFileReader:
+    """This reader claims DICOM files with higher-than default confidence and loads them directly into the scene.
+    """
+
+    def __init__(self, parent):
+        self.parent = parent
+
+    def description(self):
+        return 'DICOM direct load'
```

Comment:
Should this be marked translatable?

Reply:
Yes it appears a reader's `description` should be marked translatable per
https://github.com/Slicer/Slicer/blob/7ef59614c2458e4f693ab77834e6726cf6ae0540/Modules/Loadable/Volumes/qSlicerVolumesReader.cxx#L90-L93

I had copied based on the "DICOM import" reader which didn't translate it. I've pushed an additional commit to mark various scripted file reader's description as being translatable. @lassoan can probably confirm this action with the translation work he has been working on.
https://github.com/Slicer/Slicer/blob/7ef59614c2458e4f693ab77834e6726cf6ae0540/Modules/Scripted/DICOM/DICOM.py#L900-L901